src/libstd/ffi/mod.rs

```diff
@@ -8,7 +8,156 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-//! Utilities related to FFI bindings.
+//! This module provides utilities to handle data across non-Rust
```

Comment:
I'd keep this short summary, but with a newline between it, so you get the summary. That is:

```
///! Utilities related to FFI bindings.
//!
//! This module provides utilities....
```